assets/js/state/softwareUpdates.js

```diff
@@ -2,7 +2,7 @@ import { createAction, createSlice } from '@reduxjs/toolkit';
 import { find, get, isEmpty } from 'lodash';
 
 const initialState = {
-  settingsConfigured: false,
```

Comment:
Why did this need to be changed? :thinking:

Reply:
It's a leftover from a fix I attempted for the same behavior I think you found in #2821, for which you just added additional dispatches in the sagas. I can restore it to `false` if you want

Reply:
Yes because if it changes then we need to display a call to action for the user to go configuring the connection details